scripts/build_bug_report.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
This script's name isn't 100% accurate yet. 

We can choose to keep it or not, but I wanted a portable way to get that `GBALATRO_VERION` string, so I chose python.